packages/dnb-eufemia/src/extensions/forms/ValueBlock/ValueBlock.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import { Span } from '../../../elements'
+import { FormLabel } from '../../../components'
+import { forwardSpaceProps } from '../utils'
+import classnames from 'classnames'
+import type { ComponentProps } from '../component-types'
+import type { ValueProps } from '../value-types'
+
+export type Props = ComponentProps &
+  Omit<ValueProps<unknown>, 'value'> & {
+    children?: React.ReactNode
+  }
+
+export default function StringValue(props: Props) {
+  const {
+    className,
+    'data-testid': dataTestId,
+    path,
+    label,
+    inline,
+    placeholder,
+    showEmpty,
+    children,
+  } = props
+  if (
+    (children === undefined || children === null || children === false) &&
+    !showEmpty &&
+    !placeholder
+  ) {
+    return null
+  }
+
+  return (
+    <Span
+      className={classnames(
+        'dnb-forms-value',
```

Comment:
```suggestion
        'dnb-forms-value-block',
```

Not sure if this is really needed though.
But if we do it, we must remember to search and replace all references for `dnb-forms-value` to be `dnb-forms-value-block`.

Reply:
Agree. ValueBlock was called Value earlier, so just a miss on my part.